Source/DafnyStandardLibraries/Makefile

```diff
@@ -38,6 +38,7 @@ test-cs:
 	$(DAFNY) test $(VERIFY_OPTIONS) -t:cs examples/dfyconfig.toml examples/BoundedInts/NonDefault.cs --output:build/stdlibexamples --coverage-report:build/testcoverage
 	$(DAFNY) test $(VERIFY_OPTIONS) -t:cs examples/nondeterministic/dfyconfig.toml --output:build/stdlibexamples_nondeterministic --coverage-report:build/nondeterministic_testcoverage
 	TARGETLANG=cs make -C src/Std/TargetSpecific test
+	$(DAFNY) test $(VERIFY_OPTIONS) -t:cs --standard-libraries examples/StatisticsExamples.dfy src/Std/FileIOInternalExterns/Std/FileIOInternalExterns/ExternalMath.cs --allow-warnings --output:build/Statistics_Examples --coverage-report:build/statistics_coverage
```

Comment:
I think it would be better to add this module to the TargetSpecific set of modules, since it depends on an extern file. That doesn't mean it's a lesser library or anything, and we can easily fill in the other targets later.

You shouldn't have to actually change any makefiles then AFAICT.